### PR TITLE
[13.0][FIX] ddmrp: variables get overriden in each iteration.

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
@ForgeFlow

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

File: ddmrp/models/stock_move.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ def _update_ddmrp_nfp(self):
         # outgoing and `in_buffers` as incoming.
         out_buffers = in_buffers = self.env["stock.buffer"]
         for move in self:
-            out_buffers = move.mapped("product_id.buffer_ids").filtered(
+            out_buffers += move.mapped("product_id.buffer_ids").filtered(
```
Comment (jbaudoux) [inline ddmrp/models/stock_move.py]: I would have used |=

Comment (LoisRForgeFlow) [reply ddmrp/models/stock_move.py]: Changed

**Review by mileo — APPROVED** [review]

**Review by jbaudoux — APPROVED** [review]
LGTM

**Pull request merged**